packages/filefeeds/package.json

```diff
@@ -27,7 +27,7 @@
     "dev": "rollup -c -w",
     "build": "yarn run clean && rollup -c",
     "build-test": "parcel build test/index.html",
-    "test": "parcel serve test/index.html -p 4242 --open",
+    "test": "parcel serve test/index.html -p 4252 --open",
```

Comment:
any reason for this? or just want to be separate from importer version?

Reply:
Yep, I'm updating the port values so we have separate endpoints for each test page so that we can have them running in parallel and don't have to close the test server to switch.